ramls/usergroup.json

```
@@ -16,6 +16,10 @@
       "description": "A UUID identifying this group",
       "type": "string"
     },
+    "expirationOffset": {
```

Previously, periods of time without any units have caused confusion.

If this is always going to be measured in days, please make that explicit in the name. 

```suggestion
    "expirationOffsetInDays": {
```

---

Makes absolute sense. I changed that.